### ISSUE TITLE
Issue with "DOMMatrix()": (string containing a sequence of numbers)

### ISSUE THREAD

**Issue by cubuspl42** [mdn/content#10400]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/DOMMatrix/DOMMatrix

#### What information was incorrect, unhelpful, or incomplete?

I'm not absolutely sure about this, but I believe that in the phrase (emphasis mine)...

> **A string containing a sequence of numbers** or an array of numbers specifying the matrix you want to create, or a CSS transform string.

...the emphasized part is misleading or incorrect.

#### Specific section or headline?

[Parameters](https://developer.mozilla.org/en-US/docs/Web/API/DOMMatrix/DOMMatrix#parameters
)
#### What did you expect to see?

The [algorithm from the specs](https://drafts.fxtf.org/geometry/#dommatrix-create) indicates that the string-accepting variant of constructor expects only a string in [CSS `transform` format](https://drafts.csswg.org/css-transforms-1/#propdef-transform).

#### Did you test this? If so, how?

I've checked in Google Chrome 95.0.4638.69 that `new DOMMatrix("translate(100px, 200px)")` evaluates to...

```
{
    "a": 1,
    "b": 0,
    "c": 0,
    "d": 1,
    "e": 100,
    "f": 200,
    "m11": 1,
    "m12": 0,
    "m13": 0,
    "m14": 0,
    "m21": 0,
    "m22": 1,
    "m23": 0,
    "m24": 0,
    "m31": 0,
    "m32": 0,
    "m33": 1,
    "m34": 0,
    "m41": 100,
    "m42": 200,
    "m43": 0,
    "m44": 1,
    "is2D": true,
    "isIdentity": false
}
```

I've tried passing "a string containing a sequence of numbers", guessing the actual format:
```
new DOMMatrix("1,2,3,4,5,6")

VM260:1 Uncaught DOMException: Failed to construct 'DOMMatrix': Failed to parse '1,2,3,4,5,6'.
    at <anonymous>:1:1
```

```
new DOMMatrix("1 2 3 4 5 6")

VM330:1 Uncaught DOMException: Failed to construct 'DOMMatrix': Failed to parse '1 2 3 4 5 6'.
    at <anonymous>:1:1
```

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/dommatrix/dommatrix`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/DOMMatrix/DOMMatrix
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/dommatrix/dommatrix/index.md
* Last commit: https://github.com/mdn/content/commit/6dc80ea6ca0fc028a29a318300b21d7ac9f80978
* Document last modified: 2021-11-01T22:39:19.000Z

</details>

**Comment by himanshugarg** [comment]
Possible duplicate of now closed #10351

**Comment by cubuspl42** [comment]
> Possible duplicate of now closed #10351

You are right.
